conf_linux_public.py

```diff
@@ -101,15 +111,32 @@ def _get_api_version(repo_path):
 if args.get('api_latest'):
     cmake_command.append('-DAPI:STRING=latest')
 
+#Temporary solution for gcc-8: https://github.com/Intel-Media-SDK/MediaSDK/issues/359
+if args.get('gcc_latest'):
+    cmake_command = ['cmake',
+                     '--no-warn-unused-cli',
+                     '-Wno-dev -G "Unix Makefiles"',
+                     '-DCMAKE_C_FLAGS_RELEASE="-O2 -Wformat -Wformat-security -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong"',
+                     '-DCMAKE_CXX_FLAGS_RELEASE="-O2 -Wformat -Wformat-security -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong "',
+                    ]
+
 cmake_command.append(str(MEDIA_SDK_REPO_DIR))
 
 cmake = ' '.join(cmake_command)
 
-action('cmake',
-       cmd=f'{ENABLE_DEVTOOLSET} && {cmake}')
+if args.get('gcc_latest'):
```

Comment:
please don't duplicate action under condition here and below
create different command (or part of command) only